rust/crd/src/lib.rs

```diff
@@ -71,10 +71,9 @@ pub enum NifiVersion {
 
     // TODO: NiFi 1.14 does not work with this operator yet <https://github.com/stackabletech/nifi-operator/issues/82>
     //    Therefore we skip using it in the CRD for now. Should be uncommented as soon as #82 is fixed.
-    //#[serde(rename = "1.14.0")]
-    #[serde(skip)]
-    #[strum(serialize = "1.14.0")]
-    v1_14_0,
+    #[serde(rename = "1.15.0")]
+    #[strum(serialize = "1.15.0")]
+    v1_15_0,
```

Comment:
I think we then can remove 1.13.2 completly?

Reply:
Yes we can, done

Reply:
This is obsolete, didnt we want to keep it now :) ?